src/main/java/com/google/idea/perf/CallTree.kt

```diff
@@ -21,6 +21,7 @@ interface CallTree {
     val tracepoint: Tracepoint
     val callCount: Long
     val wallTime: Long
+    val maxCallTime: Long
```

Comment:
Note for the future: at some point we might want to measure 'CPU time' in addition to wall time, at which point we might want to rename this to 'maxWallTime'.

Reply:
Yeah, good idea. I decided to rename it as soon as possible. Maybe we should also rename `wallTime` to `totalWallTime` later on. What do you think?

Reply:
Yeah, `wallTime` or `totalWallTime` both sound good to me. Let's keep `wallTime` for now.